service/src/main/resources/application.properties

```diff
@@ -30,4 +30,4 @@ spring.cloud.stream.bindings.consumer-out-0.destination=Notification
 spring.cloud.stream.bindings.notification-in-0.destination=Notification
 spring.datasource.maxRetries=1
 spring.cache.type=redis
-
+spring.datasource.timeToLive=259200
```

Comment:
Mention the unit as well in the properties

fileStatus.cache.expiry.inseconds=

Avoid using spring.* since those have are generally for the spring boot framework